make_nf_run_script.py

```diff
@@ -0,0 +1,38 @@
+import os
+import argparse
+
+parser = argparse.ArgumentParser(description='Generate run script for nextflow pipelines')
+parser.add_argument('--project', required=True, help='Project name')
+parser.add_argument('--genome', required=True, help='Reference genome, e.g. GRCm38')
+parser.add_argument('--pipeline', required=True, help='Reference genome, e.g. methylseq')
+
+args = parser.parse_args()
+project = args.project
+genome = args.genome
+pipeline = args.pipeline
+
+analysis_path = "/proj/ngi2016001/nobackup/NGI/ANALYSIS/"
+scripts_path = os.path.join(analysis_path, project, "scripts")
+template_path = "/lupus/ngi/production/latest/sw/upps_standalone_scripts/run_script_templates"
```

Comment:
You could make this path dynamic with somethin like: 
`os.path.join(os.path.dirname(os.path.realpath(__file__))),"run_script_templates")`

Reply:
But maybe just worth it if the path to the config in the `rnaseq_GRCh38_gencode` template is dynamic too. Maybe not worth the hassle. Let me know if it's something you would like to do.

Reply:
Like you say, if it's not complete, I don't think it adds much to be honest. I thought about switching to templating like jinja for the whole thing to avoid the slightly unspecific replacements, but for now just added the minimum to adapt to the gencode annotations.